draft-ietf-acme-acme.md

```diff
@@ -2178,10 +2177,11 @@ Content-Type: application/jose+json
 }
 ~~~~~~~~~~
 
-The server updates the authorization document by updating its representation of
-the challenge with the response object provided by the client.  The server MUST
+The server MUST
```

Comment:
This seems like an unnecessarily heavy cut.  How about if we change "with" to "based on" -- the server takes the client's response, does some stuff, and updates its state.

Reply:
I decided to let the whole paragraph stand and just add the clarifying note.